sdk/@launchdarkly/observability-android/lib/src/main/kotlin/com/launchdarkly/observability/replay/capture/TiledSignature.kt

```diff
@@ -0,0 +1,100 @@
+package com.launchdarkly.observability.replay.capture
+
+import android.graphics.Bitmap
+
+data class TiledSignature(
+    val tileHashes: LongArray
+) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as TiledSignature
+
+        if (!tileHashes.contentEquals(other.tileHashes)) {
+            return false
+        }
+
+        return true
```

Comment:
Redundant if statement 

```suggestion
        return tileHashes.contentEquals((other as TiledSignature).tileHashes)
```